Modules/Core/Common/src/itkPoolMultiThreader.cxx

```diff
@@ -273,7 +281,12 @@ PoolMultiThreader ::ParallelizeImageRegion(unsigned int         dimension,
       for (ThreadIdType i = 1; i < splitCount; i++)
       {
         exceptionHandler.TryAndCatch([this, i, &reporter] {
-          m_ThreadInfoArray[i].Future.get();
+          std::future_status status;
+          do
+          {
+            status = m_ThreadInfoArray[i].Future.wait_for(threadCompletionPollingInterval);
+            reporter.Completed(0);
+          } while (status != std::future_status::ready);
           reporter.CompletedPixel();
```

Comment:
Using the TotalProgressReporter, the progress is going to be incremented by the chuck amount. The filter may  also be doing the progress, the two will be added together causing some inaccuracy/difficulties.

Reply:
What is "chuck amount"? I followed your other suggestions.

Reply:
Looks like you reverted to just the progress reported 👍